src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/vet/VetOptionsComponentObject.java

```diff
@@ -142,20 +143,11 @@ public void verifyVideoAlignmentSelected(PositionsVideo positions) {
     Assertion.assertEquals(selectedPositionId, desiredPositionId);
   }
 
-  public void clickUpdateVideo() {
-    wait.forElementVisible(updateVideoButton);
-    scrollAndClick(updateVideoButton);
-    PageObjectLogging.log("updateVideoButton", "update video button clicked", true, driver);
-  }
-
-
-  public void verifyVideoWidth(int widthDesired) {
+  public int getVideoWidth() {
     wait.forElementVisible(widthInputField);
     int width = Integer.parseInt(widthInputField.getAttribute("value"));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "width". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1488)